docs/13-walkthroughs/3-add-logging.md

```diff
@@ -1,39 +1,59 @@
 
+## System Requirements
 
-## Walkthrough Prerequisites
-1. The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
-1. Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
+- You'll need an internet connection to grab the Zarf Init Package if it's not already on your machine.
 
+## Prerequisites
 
-## Install the logging component
+Prior to this walkthrough you'll want to have a working cluster with Zarf initialized
+1.  Zarf binary installed on your $PATH: ([Installing Zarf](../3-getting-started.md#installing-zarf))
+2. [Initialize a cluster](./1-initializing-a-k8s-cluster.md).
 
-To install the logging component, follow the [Initializing a Cluster Instructions](./1-initializing-a-k8s-cluster.md), but instead answer `y` when asked to install the `logging` component
+## Walkthrough
+[![Walkthrough: Add Logging to an existing Zarf Cluster with Zarf Init](../.images/walkthroughs/logging_thumbnail.png)](https://youtu.be/FYkafs1idlQ "Add Logging to an existing Zarf Cluster with Zarf Init")
 
+1. Run the `zarf init` command on your cluster.
 
-## Note the credentials
+```sh
+zarf init
+```
 
-Review the `zarf init` command output for the following:
+2. When prompted to deploy the package select `y` for Yes, then hit the `enter` 
+key. <br/> When prompted to deploy the logging component select `y` for Yes, then hit the `enter` key.
 
-![logging-creds](../.images/walkthroughs/logging_credentials.png)
+![The zarf operator is propted to deploy the zarf package and the logging component, selecting 'y' for yes, for each of them.](../.images/walkthroughs/logging_prompts.png)
 
-You should see a section for `Logging`.  You will need these credentials later on.
+4. You can automatically accept the logging component, and confirm the package using the `--confirm` and `--components` flags.
 
+<iframe src="/docs/walkthroughs/logging_init.html" style={{
+  height: '800px',
+  width: '100%',
+  border: 'none'}}></iframe>
 
-## Deploy the Doom Walkthrough
+### Connecting to the Logging Component
+
+#### Note the Credentials
+
+1. Review the `zarf init` command output for the following:
+![logging-creds](../.images/walkthroughs/logging_credentials.png)
+You should see a section for `Logging`.  You will need these credentials later on.
 
-Follow the remainder of the [Doom Walkthrough](./2-deploying-doom.md).
+## Deploy the Doom Walkthrough
 
+1. If you haven't already, deploy the [Doom Walkthrough](./2-deploying-doom.md).
 
 ## Check the logs
 
 :::note
 
-Because Doom is freshly installed it is recommended to refresh the page a few times to generate more log traffic to view in Grafana
+Because Doom maybe freshly installed it is recommended to refresh the page a few times to generate more log traffic to view in Grafana
```

Comment:
```suggestion
Because Doom may be freshly installed it is recommended to refresh the page a few times to generate more log traffic to view in Grafana
```